.github/tests/agglayer-sp1-key.yml

```
@@ -1,10 +1,15 @@
 deployment_stages:
   deploy_cdk_bridge_ui: false
+  # RollupID for the CI would be:
+  # 0: L1 (271828)
+  # 1: CDK (10101)
+  # 2: OP (2151908)
+  # 3: attach-cdk-3 (30303)
+  # 4: attach-cdk-4 (40404)
```

I think we should find better names for these chains, I'm open to suggestions. I re-used the new polygon-cdk naming standards defined [here](https://0xpolygon.slack.com/archives/C061XGY1M96/p1737639682918159) - maybe some of the names are wrong, please double-check.

```suggestion
  # 0: L1 (271828)
  # 1: cdk-erigon-validium-1 (10101)
  # 2: cdk-opgeth-validium (2151908)
  # 3: cdk-erigon-validium-2 (30303)
  # 4: cdk-erigon-sovereign (40404)
```

We should also rename the args files and maybe move all of them to a specific directory (e.g. `chains`) under `.github/tests`.

---

I that naming makes sense, but within the workflows, I think referencing the chains by their rollup-id would be more intuitive, especially because the bridge scenarios require us to input the rollup-ids for the destination network so perhaps something like:
`cdk-1`, `cdk-2`, `cdk-3`, `cdk-4`

Then in the args file, we can maintain the comments to know which rollup-id refers to what cdk.

---

Yeah I think `cdk-{id}` makes sense, it's shorter and easier to work with! I would still add something maybe add the top of the args file to explain the purpose of the chain (e.g. op/cdk, validium/sovereign, etc.).

---

For the file names maybe it could be something like:

- `.github/tests/agglayer-sp1-key.yml` => `.github/tests/chain/l1-cdk1-cdk2.yml`
- `.github/tests/attach-cdk-3.yml` => `.github/tests/chain/cdk3.yml`
- `.github/tests/attach-cdk-4.yml` => `.github/tests/chain/cdk4.yml`